### DISCUSSION TITLE
Previous versions of casks not available anymore?

### DISCUSSION THREAD

**Discussion by stefantimm** [Homebrew/discussions#388] [Casks]
I recently reinstalled my MacBook and was able to select the casks "calibre4" and "omniplan3". This was quite useful, as calibre 5 breaks all sort of things and I don't have a license for OmniPlan 4 (yet).

Today I did a reinstall on my iMac and the casks mentioned above don't exist anymore. Is this on purpose?

What is the proper way to install the older versions. Using the hash link from GitHub also gives me an error message.

**Comment by alebcay** [comment]
For casks specifically, alternate versions of software are available in the `homebrew/cask-versions` tap. So to install the two pieces of software in particular you mentioned:
```
brew tap homebrew/cask-versions
brew install calibre4 omniplan3
```
